portfolio/src/main/webapp/script.js

```diff
@@ -53,6 +53,17 @@ function createCommentElement(comment) {
 
   commentElement.appendChild(nameElement);
   commentElement.appendChild(textElement);
+
+  imageURL = comment.imageURL;
+  if(imageURL != null){
+    console.log("Vibes");
+    const imgElement = document.createElement('IMG');
```

Comment:
nit: Tags should be lowercase.

https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
Okay sure, I've amended it.